dbms/src/main/java/org/polypheny/db/view/MaterializedViewManagerImpl.java

```diff
@@ -172,21 +174,27 @@ public synchronized void addMaterializedInfo( Long materializedId, MaterializedC
      * update candidates for materialized view with freshness updates
      *
      * @param transaction transaction of the commit
-     * @param tableNames table that was changed
+     * @param tableIds table that was changed
      */
     @Override
-    public void addTables( Transaction transaction, List<String> tableNames ) {
-        if ( tableNames.size() > 1 ) {
-            try {
-                CatalogTable catalogTable = Catalog.getInstance().getTable( 1, tableNames.get( 0 ), tableNames.get( 1 ) );
-                long id = catalogTable.id;
-                if ( !catalogTable.getConnectedViews().isEmpty() ) {
-                    updateCandidates.put( transaction.getXid(), id );
-                }
-            } catch ( UnknownTableException e ) {
-                throw new RuntimeException( "Not possible to getTable to update which Tables were changed.", e );
+    public void notifyModifiedTables( Transaction transaction, Collection<Long> tableIds ) {
```

Comment:
Is there a reason why the logic in this class is still relational. We yet only support it for relational namespaces, but are there conceptual differences beside that we do not get information on changes from the statistics.

Reply:
Actually no, we should with minor adjustment be able to enable it for all. But I would do this in a separate pull request, to add some more testing. 
Already renamed wherever sensible.